cmd/go-quai/main.go

```diff
@@ -83,16 +83,7 @@ var (
 		utils.GCModeFlag,
 		utils.SnapshotFlag,
 		utils.TxLookupLimitFlag,
-		utils.LightServeFlag,
-		utils.LightIngressFlag,
-		utils.LightEgressFlag,
-		utils.LightMaxPeersFlag,
-		utils.LightNoPruneFlag,
 		utils.LightKDFFlag,
```

Comment:
Why didn't this flag get removed too?

Reply:
That flag is not related to the light chain. its to reduce the RAM and CPU usage